tests/v2prov/operations/etcdsnapshot.go

```diff
@@ -182,7 +183,8 @@ func RunSnapshotCreateTest(t *testing.T, clients *clients.Clients, c *v1.Cluster
 			var snapshots []*rkev1.ETCDSnapshot
 			// Parse the snapshot time from the snapshot file name
 			for _, s := range snapshotsList.Items {
-				if s.SnapshotFile.NodeName == targetNode && s.SnapshotFile.Size > 0 {
+				// 1.32.4, 1.31.8, and 1.30.12 onwards set the nodename correctly, so use the annotation to determine the target node.
+				if (s.SnapshotFile.NodeName == targetNode || (targetNode == "s3" && s.Annotations[snapshotbackpopulate.StorageAnnotationKey] == string(snapshotbackpopulate.S3))) && s.SnapshotFile.Size > 0 {
```

Comment:
This can only be changed when 1.33.0 is the lowest supported k8s version.

Reply:
Maybe add a TODO note to remove it once 1.32 is EOL?